### PR TITLE
Fetching Community Blog Posts

### PR DESCRIPTION
**Pull request by coffeeboo**
currently only attempts to print it, and returns error
"Unsupported service. Your client may be out of date. Please update it to the latest version."

tons of problems and not ready to merge to version 2... yet.

### CODE REVIEW

**Comment by moelrobi** [comment]
Looking at it soon.

File: helpers/endpoints.js [hunk 1]
```diff
@@ -6,5 +6,6 @@ module.exports = {
     getComs: prefix + '/v1/g/s/community/joined?start=0&size=50',
     getJoinedChats: (com) => `${prefix}/v1/${com}/s/chat/thread?type=joined-me&start=0&size=100`,
     loadChat: (com, uuid, count) => `${prefix}/v1/${com}/s/chat/thread/${uuid}/message?start=0&size=${count}&cv=v1.2`,
-    sendChat: (com, uuid) => `${prefix}v1/${com}/s/chat/thread/${uuid}/message`
+    sendChat: (com, uuid) => `${prefix}v1/${com}/s/chat/thread/${uuid}/message`,
+    getCommunityBlogFeed: (com, startAt, size) => `${prefix}/${com}/s/feed/blog-all?start=${startAt}&size=${size}`,
```
Comment (moelrobi) [inline helpers/endpoints.js]: ${prefix}/v1/${com}/s/feed/blog-all?start=${startAt}&size=${size}

Fixed

Comment (coffeeboo) [reply helpers/endpoints.js]: Thanks!

**Comment by felixfong227** [comment]
Will take a look

**Comment by moelrobi** [comment]
Is it still broken? 😕

**Comment by coffeeboo** [comment]
@moelrobi I will fix it on my free time, currently in school

**Pull request merged**